### ISSUE TITLE
What is the training strategy for PASCAL VOC?

### ISSUE THREAD

**Issue by SkeletonOne** [xuannianz/keras-fcos#14]
Hi, thanks for your excellent work.

I'd like to know about the input size for training, the lr and the lr decay. Thanks!

**Comment by nashurgessa** [comment]
When I train in my own data, the loss shows me, nan. Any suggestion regarding this?
